### ISSUE TITLE
Call.js error

### ISSUE THREAD

**Issue by SmellySmel** [Ross1337/SMSBotBypass#67]
My twilio account is upgraded so i filled the required information in the config.js file
and read about this error from you guys then i went to call.js and replaced the line 122 with my twilio number
and im still getting the same error any help guys?

**Comment by Ross1337** [comment]
Hello, use insomnia to make a test call on your api (route localhost:1337/call) with these parameters
![image](https://user-images.githubusercontent.com/45340378/134648532-1a3d57fb-5a66-472b-92f7-dcc1a404771d.png)

Twilio will show you where does the error came from (npm start -> test your route -> show the error)

**Comment by SmellySmel** [comment]
can you send me link to insomnia also i dont know what that is

**Comment by Ross1337** [comment]
https://insomnia.rest/

**Comment by johnwick9t7** [comment]
![Screenshot_445](https://user-images.githubusercontent.com/92142631/136504540-479935e3-3593-4ad4-926f-39a0f8e390b8.png)

same issues bro, do i have to upgrade my account strictly?

**Comment by darkhacknet** [comment]
> ![Screenshot_445](https://user-images.githubusercontent.com/92142631/136504540-479935e3-3593-4ad4-926f-39a0f8e390b8.png)
> 
> same issues bro, do i have to upgrade my account strictly?

Yes of course you need tonuogratenthat Account bro.. Check the readme..
